Packs/MajorBreachesInvestigationandResponse/ReleaseNotes/1_6_2.md

```diff
@@ -0,0 +1,4 @@
+
+#### Playbooks
+##### Codecov Breach - Bash Uploader
+- Added code search using GitHub to find any use of Codecov bash uploader in the repositories.
```

Comment:
Added task to find any use of Codecov bash uploader in specified Github repositories.